packages/tools/playground/src/components/rendererComponent.tsx

```diff
@@ -3,7 +3,7 @@ import type { GlobalState } from "../globalState";
 import { RuntimeMode } from "../globalState";
 import { Utilities } from "../tools/utilities";
 import { DownloadManager } from "../tools/downloadManager";
-import { Engine, WebGPUEngine } from "@dev/core";
+import { Engine, EngineStore, WebGPUEngine } from "@dev/core";
 
 import type { Nullable } from "@dev/core";
 import type { Scene } from "@dev/core";

```

Comment:
```suggestion
import type { Nullable, Scene } from "@dev/core";
```

Done just too force the CI to re-run (got stuck for some reason)